library/alloc/benches/slice.rs

```diff
@@ -218,7 +218,7 @@ fn gen_strings(len: usize) -> Vec<String> {
     let mut v = vec![];
     for _ in 0..len {
         let n = rng.gen::<usize>() % 20 + 1;
-        v.push((&mut rng).sample_iter(&Alphanumeric).take(n).collect());
+        v.push((&mut rng).sample_iter(&Alphanumeric).map(|byte| byte as char).take(n).collect());
```

Comment:
I think `sample_string` is the function meant to be used for this now

Reply:
Thanks, done.